dotCMS/src/main/java/com/dotmarketing/filters/CMSFilter.java

```diff
@@ -149,7 +150,7 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
             return;
         }
 
-        if (iAm == IAm.FILE) {
+        else if (iAm == IAm.FILE) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)